src/lxc/conf.h

```diff
@@ -248,6 +248,11 @@ enum lxchooks {
 
 extern char *lxchook_names[NUM_LXC_HOOKS];
 
+struct lxc_state_client {
```

Comment:
Can you explain in the commit message for future readers why the lxc_state_client gets moved from handler to conf?

Reply:
Yeah, good idea.

Reply:
done